lib/views/location_card.dart

```diff
@@ -0,0 +1,130 @@
+                child: Column(
+                  children: [
+                    const SizedBox(height: 20.0),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: [
+                        Text(
+                          widget.location.location,
+                          style: const TextStyle(
+                            letterSpacing: 2.0,
+                            color: Colors.white,
+                          ),
+                        ),
+                      ],
+                    ),
+                    const SizedBox(height: 20.0),
+                    widget.location.time != null
+                        ? Text(
+                            widget.location.time ?? '',
+                            style: TextStyle(
+                              fontSize: 66.0,
+                              color: Colors.white,
+                            ),
+                          )
+                        : const SpinKitSpinningLines(
+                            color: Colors.blue,
+                            size: 80.0,
+                          ),
+                    Expanded(
+                      child: Center(
+                        child: ListView(
+                          scrollDirection: Axis.horizontal,
+                          padding: const EdgeInsets.all(8),
+                          children: <Widget>[
+                            Container(
+                              width: 200,
+                              child: Align(
+                                alignment: Alignment.topCenter,
+                                child: Text(
+                                  'Day of the week: ${widget.location.dayOfWeek.toString()}',
+                                  style: TextStyle(
+                                    color: Colors.white,
+                                  ),
+                                ),
+                              ),
+                            ),
+                            Container(
+                              width: 200,
+                              child: Text(
+                                'Day of the year: ${widget.location.dayOfyear.toString()}',
+                                style: TextStyle(
+                                  color: Colors.white,
+                                ),
+                              ),
+                            ),
+                            Container(
+                              width: 200,
+                              child: Text(
+                                'Unix time: ${widget.location.unixTime.toString()}',
+                                style: TextStyle(
+                                  color: Colors.white,
+                                ),
+                              ),
+                            ),
+                            Container(
```

Comment:
These Containers are all pretty similar. Maybe factor them out into a `DetailsCard` widget, that only takes the parts that a different (e.g. a `title` and a `value`) as parameters?